app/(golems)/golems/skills/[name]/page.tsx

```
@@ -358,6 +358,243 @@ export default async function SkillDetailPage({
+                  className="inline-flex min-h-[44px] items-center gap-1.5 text-sm text-[#a89078] transition-colors hover:text-[#c0b8a8]"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  View source on GitHub
+                  <svg
+                    className="h-4 w-4"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                    strokeWidth={2}
+                  >
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)